composer.lock

```diff
@@ -0,0 +1,3836 @@
+{
+    "_readme": [
```

Comment:
This file should be excluded (is so in gitignore already afaik)

Reply:
I would not ignore the composer.lock file, especially if there is no problem with it.

Reply:
There is. also for ci testing. Please remove it.

Reply:
Then please explain what the current problem is.

Reply:
We discussed that many times before for non project (libraries) packages.
Here also composer install will not properly pull latest (which it must!)